eventually/authentication/models.py

```diff
@@ -0,0 +1,145 @@
+        7. updated_at: DateTimeField
+
+    """
+
+    first_name = models.CharField('first_name', max_length=20)
+    middle_name = models.CharField('middle_name', max_length=20)
+    last_name = models.CharField('second_name', max_length=20)
+    email = models.EmailField('email_address', max_length=40, unique=True)
+    password = models.CharField('password', max_length=128)
+    updated_at = models.DateTimeField(auto_now=True)
+    created_at = models.DateTimeField(auto_now_add=True, editable=False)
+
+    def __str__(self):
+        """
+            This magic method is redefined to show first, last name and id of the CustomUser object.
+        """
+        return '{} {} {}'.format(self.first_name, self.last_name, self.id)
+
+    @staticmethod
+    def get_by_id(user_id):
+        """
+        :param user_id: SERIAL: the id of a user to be found in the DB
+        :return: user object or None if a user with such ID does not exist
+        """
+        try:
+            return CustomUser.objects.get(id=user_id)
+        except CustomUser.DoesNotExist:
+            return None
+
+    @staticmethod
+    def delete_by_id(user_id):
+        """
+        :param user_id: an id of a user to be deleted
+        :return: True if object existed in the db and was removed or False if it didn't exist
+        """
+        try:
+            user = CustomUser.objects.get(id=user_id)
+            user.delete()
+            return True
+        except AttributeError:
+            return False
+
+    @staticmethod
+    def create(first_name, middle_name, last_name, email, password):
+        """
+        :param first_name: first name of a user
+        :param middle_name: middle name of a user
+        :param last_name: last name of a user
+        :param email: email of a user
+        :param password: password of a user
+        :return: a new user object which is also written into the DB
+        """
+        user = CustomUser(first_name=first_name,
+                          last_name=last_name,
+                          middle_name=middle_name,
+                          email=email,
+                          password=password)
+        try:
+            user.save()
+        except CustomUser.OperationalError():
+            return False
+        return user
+
+    def myattr_to_dict(self):
+        """
+        :return: manually-defined attributes of the object. Inherited attributes aren't shown.
+        :example: {'id': 8, 'first_name': 'fn', 'middle_name': 'mn',
+            'last_name': 'ln', 'email': 'ln@mail.com', 'password': '2',
+            'created_at': '1509393504', 'updated_at': '1509402866'}
+
+        """
+        return {
+            'id': self.id,
+            'first_name': self.first_name,
+            'middle_name': self.middle_name,
+            'last_name': self.last_name,
+            'email': self.email,
+            'password': self.password,
+            'created_at': format(self.created_at, u'U'),
```

Comment:
does it work right?

Reply:
Yes. Here's the example of the output: 'created_at': '1509393504', 'updated_at': '1509402866'

Reply:
new Date(1509393504)
Sun Jan 18 **1970** 13:16:33 GMT+0200 (FLE Standard Time)

Reply:
Fixed.
datetime.datetime.fromtimestamp(1509535921)
datetime.datetime(2017, 11, 1, 11, 32, 1)